changelog

```diff
@@ -1,3 +1,193 @@
+    * Engine bugs:
+      * Fix edge case in code style checker. (@tibetiroka)
+      * Don't consider "Parsing: <file>" output when in debug mode as errors when test parsing. (@warp-core)
+      * Fix game crash on short error messages. (@tibetiroka)
+      * Don't draw name tooltips for ships hidden under the shop sidebar footer. (@TomGoodIdea)
+      * Variant ships correctly inherit display names from base ships. (@TomGoodIdea)
+      * Use singular "ship" in messages when recalling/deploying a single ship. (@tibetiroka)
+      * Fix errors in the copyright file. (@tibetiroka)
+      * Rename variable to avoid ambiguity with std::queue in some versions of clang. (@quyykk)
+      * Fix ship parenting on takeoff with fighter flagship. (@tibetiroka)
+      * Correctly handle arrow keys with no item selected in the shop. (@TomGoodIdea)
+      * Align plugin checkbox clickzones correctly. (@warp-core)
+      * Fix mission distance calculations with different settings. (@warp-core)
+      * Improvements to fix crashes when loading/unloading images. (@quyykk)
+      * Fix custom port button name when using "add port". (@warp-core)
+      * Corrected the parsing of the initial player repuptation grandchild node in Government. (@xxxyyyqqq12345)
+      * Draw distance to the selected system after mission markers. (@TomGoodIdea)
+      * Ships with "disables" personality ask for help disabling, instead of destroying, their enemies. (@TomGoodIdea)
+      * Suppress missions require landing in the next system messages from failed missions. (@OcelotWalrus)
+      * Remove outdated watchdog. (@petervdmeer)
+      * Properly check if a mission is failed. (@TomGoodIdea)
+      * Hardpoint labels and lines in the ship info panel will properly align to the "weapons" box from the interface definition. (@Zitchas)
+  * Game content:
+    * New content:
+      * Add a mission transporting a Hai to Unfettered space. (@roadrunner56)
+      * add Unfettered-related Hai news. (@Hurleveur)
+      * Add missions about Hogshead referencing Boaty McBoatface story. (@Saugia)
+      * Add Coalition culture conversations. (@Arachi-Lover)
+      * Add new human merchant hails about mining asteroids. (@bene_dictator)
+      * Create more passenger jobs requiring keystones in Hai space. (@roadrunner56)
+      * Add more human news referring to Korath attacks in the Core. (@roadrunner56)
+      * Conspiracy theorist spaceport news. (@tibetiroka)
+      * Add logbook entries for Alastair Korban. (@bene_dictator)
+      * A conversation about Kraz and Delta V outfits being stocked in the northern Dirt Belt and near Earth after the war begins. (@bene_dictator)
+      * A short set of missions involving the Wanderers and Mereti. (@Quantumshark)
+      * More new human planets and stations, update Kraken Station landing image. (@bene_dictator)
+      * Add new uninhabited landable planets to existing systems in Human space. (@Quantumshark)
```

Comment:
```suggestion
      * Update Kraken Station landing image. (@bene_dictator)
      * Add new planets and stations to Human space. (@bene_dictator, @Quantumshark)
```
For the changelong, I'd say it makes more sense to group these by content rather than by PR.